integration_tests/test_rough_align.py

```diff
@@ -0,0 +1,382 @@
+        for d in ROUGH_MONTAGE_TILESPECS_JSON]
+    return tilespecs
+
+@pytest.fixture(scope='module')
+def tspecs():
+    tilespecs = [renderapi.tilespec.TileSpec(json=d)
+                for d in ROUGH_DS_TEST_TILESPECS_JSON]
+    return tilespecs
+
+# A stack with multiple sections montaged
+@pytest.fixture(scope='module')
+def montage_stack(render,tspecs_from_json):
+    test_montage_stack = 'input_montage_stack'
+    renderapi.stack.create_stack(test_montage_stack, render=render)
+    renderapi.client.import_tilespecs(test_montage_stack, 
+                                      tspecs_from_json, 
+                                      render=render)
+    renderapi.stack.set_stack_state(test_montage_stack, 
+                                    'COMPLETE', 
+                                    render=render)
+
+    # assure stack is built correctly
+    assert len({tspec.tileId for tspec
+                in tspecs_from_json}.symmetric_difference(set(
+                    renderapi.stack.get_stack_tileIds(
+                        test_montage_stack, render=render)))) == 0
+    
+    zvalues = render.run(renderapi.stack.get_z_values_for_stack,
+                          test_montage_stack)
+    zs = [1020, 1021, 1022]
+    assert(set(zvalues) == set(zs))
+
+    yield test_montage_stack
+    #renderapi.stack.delete_stack(test_montage_stack, render=render)
+
+@pytest.fixture(scope='module')
+def one_tile_montage(render, tspecs):
+    one_tile_montage_stack = 'one_tile_montage_stack'
+    renderapi.stack.create_stack(one_tile_montage_stack, render=render)
+    renderapi.client.import_tilespecs(one_tile_montage_stack, 
+                                      tspecs, 
+                                      render=render)
+    renderapi.stack.set_stack_state(one_tile_montage_stack, 
+                                    'COMPLETE', 
+                                    render=render)
+    # assure stack is built correctly
+    assert len({tspec.tileId for tspec
+                in tspecs}.symmetric_difference(set(
+                    renderapi.stack.get_stack_tileIds(
+                        one_tile_montage_stack, render=render)))) == 0
+    yield one_tile_montage_stack
+
+
+@pytest.fixture(scope='module')
+def downsample_sections_dir(montage_stack, tmpdir_factory):
+    image_directory = str(tmpdir_factory.mktemp('rough_align'))
+    print(render_params)
+    ex = {
+        "render": render_params,
+        "input_stack": montage_stack,
+        "image_directory": image_directory,
+        "imgformat": "png",
+        "scale": 0.1,
+        "minZ": 1020,
+        "maxZ": 1022,
+    }
+    ex['output_json'] = os.path.join(image_directory, 'output.json')
+
+    mod = RenderSectionAtScale(input_data=ex, args=[])
+    mod.run()
+
+    out_dir = os.path.join(image_directory, render_params['project'], montage_stack, 'sections_at_0.1/001/0')
+    assert(os.path.exists(out_dir))
+    
+    files = glob.glob(os.path.join(out_dir, '*.png'))
```

Comment:
should assert that it made the right number of files for the zrange

Reply:
will add it.